pkg/zsio/zeek/flatten.go

```diff
@@ -0,0 +1,75 @@
+package zeek
```

Comment:
We'll want something like this for table and text outputs, can we put it somewhere else where its more obvious that it can be reused?

Reply:
sure

Reply:
Flattening is a property of the zeek format constraints and belongs in this module that table and text can import.